shared/elements/gaia_grid/js/grid_view.js

```diff
@@ -10,7 +10,7 @@
   /* The time for which we'll disable launching other icons after tapping on
    * an app icon. This will also disable edit mode.
    */
-  const APP_LAUNCH_TIMEOUT = 3000;
+  const APP_LAUNCH_TIMEOUT = 1000;
```

Comment:
are we dispatching the home event to apps? 

(might go away anyway if I remove the setVisible delay)